src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+            return argSplit;
+        }
+
+        return new String[] { argSplit[0], "" };
+    }
+
+    /**
+     * Executes the add task command. Adds a task depending on type.
+     *
+     * @param argument The argument from getCommandAndArgument(<string>).
+     * @param taskType The type of task.
+     */
+    private static void executeAddTask(String argument, char taskType) {
+        Task task = null;
+
+        String[] descriptionAndArg;
+        switch (taskType) {
+        case Task.TYPE_TODO:
+            task = new Todo(argument);
+            break;
+
+        case Task.TYPE_DEADLINE:
+            descriptionAndArg = getTaskDescriptionAndArg(argument,TASK_DEADLINE_SPLITTER);
+            if (descriptionAndArg[1].isEmpty()) {
+                printMessage(String.format(MESSAGE_FORMAT_DEADLINE_USAGE, COMMAND_DEADLINE, TASK_DEADLINE_SPLITTER));
+            } else {
+                task = new Deadline(descriptionAndArg[0], descriptionAndArg[1]);
+            }
+            break;
+
+        case Task.TYPE_EVENT:
+            descriptionAndArg = getTaskDescriptionAndArg(argument,TASK_EVENT_SPLITTER);
+            if (descriptionAndArg[1].isEmpty()) {
+                printMessage(String.format(MESSAGE_FORMAT_EVENT_USAGE, COMMAND_EVENT, TASK_EVENT_SPLITTER));
+            } else {
+                task = new Event(descriptionAndArg[0], descriptionAndArg[1]);
+            }
+            break;
+        }
+
+        if (task != null) {
+            TASKS.add(task);
+            printMessage(String.format(MESSAGE_FORMAT_TASK_ADDED, task.getListEntryString(), TASKS.size()));
+        }
+    }
+
+    /**
+     * Executes a command.
+     *
+     * @param command The command from getCommandAndArgument(<string>).
+     * @param argument The argument from getCommandAndArgument(<string>).
+     */
+    private static void executeCommand(String command, String argument) {
+        switch (command) {
+        case COMMAND_LIST:
+            executeListCommand();
+            break;
+
+        case COMMAND_DONE:
+            executeDoneCommand(argument);
+            break;
+
+        case COMMAND_TODO:
+            executeAddTask(argument, Task.TYPE_TODO);
+            break;
+
+        case COMMAND_DEADLINE:
+            executeAddTask(argument, Task.TYPE_DEADLINE);
+            break;
+
+        case COMMAND_EVENT:
+            executeAddTask(argument, Task.TYPE_EVENT);
+            break;
+
+        default:
+            printMessage(MESSAGE_UNKNOWN_COMMAND);
+            break;
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printMessage(MESSAGE_WELCOME);
+
+        boolean run = true;
```

Comment:
Suggestion:
Change name of "run" to "isRunning" to make it more obvious that it is a boolean